packages/codegen-ui-golden-files/lib/react/forms/datastore-create-form-with-has-one-relationship/schema.ts

```diff
@@ -14,4 +14,180 @@
   limitations under the License.
  */
 
-// TODO: implement StudioForm and DataStore schema
+import { StudioForm } from '@aws-amplify/codegen-ui';
+import { Schema } from '@aws-amplify/datastore';
+
+export const bookCreateForm: StudioForm = {
+  name: 'BookCreateForm',
+  dataType: { dataSourceType: 'DataStore', dataTypeName: 'Book' },
+  formActionType: 'create',
+  fields: {
+    primaryAuthor: {
```

Comment:
would this be implied for autogen?

Reply:
Since there are overrides for the valueMappings, there would be a StudioForm stored in the CP